include/seqan/arg_parse/arg_parse_doc.h

```diff
@@ -729,6 +733,44 @@ inline void _addDefaultValues(std::string & text, ArgParseOption const & opt)
 }
 
 // ----------------------------------------------------------------------------
+// Function _seperateExtensionsForPrettyPrinting()
+// ----------------------------------------------------------------------------
+
+inline void _seperateExtensionsForPrettyPrinting(std::vector<std::string> & file_ext,
+                                                 std::vector<std::string> & comp_ext,
+                                                 std::vector<std::string> const & validValues)
+{
+    // seperate file extensions and compression extensions
+    for (std::vector<std::string>::size_type i = 0; i < validValues.size(); ++i)
+    {
+        std::regex rgx("^(\\.)?([A-z]+)(\\.)?([A-z]+)?");
+        std::smatch result;
+
+        std::regex_search(validValues[i], result, rgx);
+
+        if (!result[4].str().empty())
+        {
+            comp_ext.push_back(result[4].str());
+            file_ext.push_back("." + result[2].str() + "[.x]");
```

Comment:
since .x could actually be a valid extension, it is not immediately clear that it is meant as a placeholder. Could we use `.?` or `.*` or `.#` instead?